README.md

```diff
@@ -452,6 +454,39 @@ Doing so, we're guaranteed to never exceed the per-shard read througput limit of
 2MB/s, while being close to it on data peaks. This has the added advantage of
 reducing the number of IO syscalls.
 
+## Working with baker.Record
+
+`baker.Record` is an interface which provides an abstraction over a record of 
+flattened data, where columns of fields are indexed through integers.
+
+At the moment, `baker` proposes an unique `Record` implementation, `baker.LogLine`.
+
+### `baker.LogLine` CSV record
+
+`baker.LogLine` is an highly optimized CSV compatible Record implementation. It 
+supports any single-byte field separator and doesn't handle quotes (neither 
+single nor double). The maximum number of fields is hard-coded by the
+`LogLineNumFields` constant which is 3000. 100 extra fields can be stored at
+runtime in a `LogLine` (also hardcoded with `NumFieldsBaker`), these extra fields
+are a fast way to exchange between filters and/or outputs but they are neither
+handled during `Parsing` (i.e `LogLine.Parse`) nor serialization (`LogLine.ToText`).
+
+If the hardcoded values for `LogLineNumFields` and `NumFieldsBaker` do not suit
+your needs, it's advised that you copy `logline.go` in your project and modify
+the constants declared at the top of the file. Your specialized `LogLine` will 
```

Comment:
the user should copy the logline but also rename the object, if I'm not mistaken. If true, I'd also change the code example below using something like `MyCustomLogline`

Reply:
No it's not necessary since the user will do this in its own project so in a different package so the definitions are never going to clash